.azure/templates/build-config-user.yml

```diff
@@ -46,7 +46,7 @@ jobs:
 
   - task: Cache@2
     inputs:
-      key:  '"${{ parameters.platform }}_${{ parameters.arch }}_${{ parameters.tls }}_${{ parameters.extraName }}_5" | .gitmodules'
+      key:  '"${{ parameters.platform }}_${{ parameters.arch }}_${{ parameters.tls }}_${{ parameters.extraName }}_${{ parameters.config }}_5" | .gitmodules'
```

Comment:
You could reset the number at the end now, right?

Reply:
Yeah I can.